html/controls/operator.js

```diff
@@ -1006,13 +1009,15 @@ function createPeriodDialog() {
 function createJamDialog() {
 	var dialog = $("<div>").addClass("NumberDialog");
 	var tableTemplate = $("<table>").addClass("Period");
-	var headers = $("<tr><td/><td/><td/><td/><td/></tr>").appendTo(tableTemplate);
+	var headers = $("<tr><td/><td/><td/><td/><td/><td/></tr>").appendTo(tableTemplate);
 	$("<a>").text("Nr").addClass("Title")
 		.appendTo(headers.find("td:eq(0)").addClass("Title"));
 	$("<a>").text("Points").addClass("Title")
 		.appendTo(headers.find("td:eq(1)").addClass("Title"));
 	$("<a>").text("Duration").addClass("Title")
 		.appendTo(headers.find("td:eq(2)").addClass("Title"));
+	$("<a>").text("PC at end").addClass("Title")
+	.appendTo(headers.find("td:eq(3)").addClass("Title"));
```

Comment:
prefer children to find when you can, it's faster.

Reply:
While I was at it I did some other straightforward optimizations as well, like using k.Period etc.